docs/en/deploy/Running-Alluxio-On-Kubernetes.md

```diff
@@ -310,6 +312,25 @@ Uninstall Alluxio as follows:
 $ helm delete alluxio
 ```
 
+#### Format Journal
+
+The helm chart contains one Kubernetes [Job](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/)
+template that can be used for formatting the Alluxio journal. The template contains one Job for each Alluxio master.
+Each Job runs `alluxio formatJournal` and formats the journal for that master.
+
+You can manually trigger the journal formatting by applying the Job template.
+```console
+# Generate the YAML definition for the journal formatting Job(s)
+$ helm template --name alluxio helm-chart/alluxio/ --set journal.format.runFormat=true -x templates/job/format-journal-job.yaml -f ./config.yaml > alluxio-format-journal-job.yaml
+# Create the Jobs(s) with kubectl
+$ kubectl create -f alluxio-format-journal-job.yaml
```

Comment:
Is it acceptable to use `kubectl` in helm section?

Reply:
this will not be needed if we suggest `helm install`